features/keychain/module/crypto/schnorr-z.js

```diff
@@ -0,0 +1,63 @@
+import { hashSync } from '@exodus/crypto/hash'
+import { hmacSync } from '@exodus/crypto/hmac'
```

Comment:
```suggestion
import { hash } from '@exodus/crypto/hash'
import { hmac } from '@exodus/crypto/hmac'
```

And turn everything into async

Reply:
https://github.com/ExodusMovement/exodus-oss/pull/185/commits/a72f683cd02ebcd6a0b265b61955cd2a4c67b672 .